cogs/example.py

```diff
@@ -1,18 +1,34 @@
 import discord
 from discord.ext import commands
 
-class Example(commands.Cog):
-    def __init__(self, bot):
-        self.bot = bot
+from cogs.base import BaseCog
+
+
+class Example(BaseCog):
+
+    def __init__(self, bot) -> None:
         self._last_member = None
+        super().__init__(bot)
 
     @commands.Cog.listener()
     async def on_member_join(self, member):
+        """
+        HuskieBot will welcome new users to the server
+
+        Parameters
+        ----------
+        member : discord.Member
+
+        Returns
+        -------
+        str
+            HuskieBot's greeting
+        """
         channel = member.guild.system_channel
         if channel is not None:
             await channel.send('Welcome {0.mention}.'.format(member))
 
-    @commands.command()
+    @commands.command(hidden=True)
```

Comment:
I will adjust the the hidden to the cog level for that. I feel we shouldn't show the example because they are just examples for the devs. I do want to convert 2 of the examples to actual commands though.